app/assets/stylesheets/modules/topbar.scss

```diff
@@ -4,18 +4,20 @@
   background: $topbar-bg-color;
 
   .name h1 {
+    position: absolute;
```

Comment:
Properties should be ordered color, display, font-family, padding, position, text-align, width